app/main.py

```diff
@@ -1,2 +1,64 @@
+from typing import Any, Union
+from copy import copy
+
+
 class Dictionary:
-    pass
+    def __init__(self) -> None:
+        self.length = 0
+        self.hash_size = 8
+        self.hash_table = [[] for _ in range(self.hash_size)]
+        self.threshold = int(self.hash_size * 2 / 3)
+
+    def resize_hash(self) -> None:
+        old_hash_table = copy(self.hash_table)
```

Comment:
simple copy will not help
try:
l1 = [[1, 2, 3], [4, 5, 6]]
l2 = copy(l1)
l2[0][1] = 10
What you expect from l1?

Reply:
Read about deepcopy